lib/passport-wsfed-saml2/samlp.js

```diff
@@ -109,11 +110,16 @@ Samlp.prototype = {
   getSamlRequestParams: function (opts, callback) {
     var options = xtend(opts || {}, this.options);
 
+    var idpUrl = options.identityProviderUrl;
+    if (typeof idpUrl !== 'string' || !validUrl.isWebUri(idpUrl)) {
```

Comment:
We can just use the `isWebUri` function right away (🤔 any reason not to do it?)
```
> vaidUrl.isWebUri(null)
undefined
> vaidUrl.isWebUri('')
undefined
> vaidUrl.isWebUri(undefined)
undefined
> vaidUrl.isWebUri({})
undefined
> vaidUrl.isWebUri('http://site.com/path')
'http://site.com/path'
```

Reply:
💯 Good catch!

Reply:
Actually, I remember why: It'd blow up with an integer...

https://github.com/gnarea/auth0-passport-wsfed-saml2/blob/0de81eb1bf7e91f96faab903339a4b631ea142cf/test/samlp.tests.js#L516

Reply:
😆 awesome